cmd/pinniped-proxy/src/main.rs

```diff
@@ -17,17 +18,26 @@ async fn main() -> Result<()> {
     pretty_env_logger::init();
     let opt = cli::Options::from_args();
 
+    // Load the default certificate authority data on startup once.
+    let default_ca_data = fs::read_to_string(opt.default_ca_cert.clone())
+        .with_context(|| format!("error loading default-ca-cert at {}", opt.default_ca_cert))?;
```

Comment:
`.with_context()` is a feature of the `anyhow` error library which allows adding some context to a result before using the `?` operator, so that the error on the cmdline looks like:
```
Error: unable to load default-ca-cert at /var/run/secrets/kubernetes.io/serviceaccount/ca.crt

Caused by:
    No such file or directory (os error 2)
```

rather than just:
```
No such file or directory (os error 2)
```